.github/dependabot.yml

```
@@ -0,0 +1,116 @@
+version: 2
+
+updates:
+  - directory: /
+    package-ecosystem: cargo
+    schedule:
+      interval: daily
+    rebase-strategy: auto
```

I'd actually do
```suggestion
    rebase-strategy: disabled
```
Everywhere, because otherwise everytime `main` changes you get a crazy amount of Github Actions triggered and that'll be quite expensive. Folks can easily do `@dependabot rebase` manually when  needed?